DetectorDescription/DDCMS/interface/DDFilteredView.h

```diff
@@ -142,11 +141,50 @@ namespace cms {
       return (dynamic_cast<Shape*>(currVol->GetShape()));
     }
 
+    // Shape of current node
+
     template <class Shape>
     bool isA() const {
       return dd4hep::isA<Shape>(solid());
     }
 
+    template <typename... Ts>
+    auto isABox(Ts&&... ts) const -> decltype(isA<dd4hep::Box>(std::forward<Ts>(ts)...)) {
+      return isA<dd4hep::Box>(std::forward<Ts>(ts)...);
+    }
```

Comment:
@ianna We currently have code that calls `isABox();` with no parameters. Do you know if this code will still work? It looks like `dd4hep::isA<dd4hep::Box>()` now requires a `const Handle<TGeoShape>& solid)` parameter.

Reply:
> @ianna We currently have code that calls `isABox();` with no parameters. Do you know if this code will still work?

Yes, it does.